tests/runtime/engine/runner.py

```diff
@@ -70,6 +70,7 @@ class Runner(object):
     SKIP_AOT_NOT_SUPPORTED = 7
     SKIP_KERNEL_VERSION_MAX = 8
     SKIP_NOT_IN_ALLOWLIST = 9
+    SKIP_LLVM_VERSION_MIN = 9
```

Comment:
Should use a different number?
```suggestion
    SKIP_LLVM_VERSION_MIN = 10
```